test/python/test_tools_database_import.py

```diff
@@ -200,3 +200,26 @@ def test_load_data(dsn, src_dir, place_row, placex_table, osmline_table, word_ta
 
     assert temp_db_cursor.table_rows('placex') == 30
     assert temp_db_cursor.table_rows('location_property_osmline') == 1
+
+@pytest.mark.parametrize("languages", (False, True))
+def test_create_country_names(temp_db_conn, temp_db_cursor, def_config,
+                              temp_db_with_extensions, monkeypatch, languages):
+    if languages:
+        monkeypatch.setenv('NOMINATIM_LANGUAGES', 'fr,en')
+    temp_db_cursor.execute("""CREATE FUNCTION make_standard_name (name TEXT)
+                                  RETURNS TEXT AS $$ SELECT 'a'::TEXT $$ LANGUAGE SQL
+                               """)
+    temp_db_cursor.execute("""CREATE OR REPLACE FUNCTION getorcreate_country(lookup_word TEXT,
+                                               lookup_country_code varchar(2))
+                            RETURNS INTEGER
+                            AS $$
+                            BEGIN
+                                INSERT INTO country_name VALUES (5, lookup_word);
+                                RETURN 5;
+                            END;
+                            $$
+                            LANGUAGE plpgsql;
+                               """)
+    temp_db_cursor.execute('CREATE TABLE country_name (id int, country_code varchar(2), name hstore)')
```

Comment:
Again, this hits all lines but doesn't really test if the language selector works. You should add a couple of lines in that table.